android/assets/html-es/scanning.html

```diff
@@ -6,11 +6,11 @@
 <link href="../style.css" rel="stylesheet" type="text/css"/>
 </head>
 <body>
-<p> Barcode Scanner escanea continuamente una región cuadrada que aparece en su pantalla - Sólo basta introducir el código de barras es completamente dentro del rectángulo del visor: </p>
+<p> Barcode Scanner escanea continuamente una región cuadrada que aparece en su pantalla - Sólo basta introducir el código de barras completamente dentro del rectángulo del visor: </p>
 <p class="imgcenter"><img src="../images/demo-yes.png" style="padding:5px"/><img src="../images/demo-no.png" style="padding:5px"/></p>
 <p> Códigos de barras 1D, como las que se encuentran en los productos requieren un teléfono con enfoque automático. Sin él, sólo los códigos QR y códigos Data Matrix serán susceptibles de ser analizados. </p>
 <p> Cuando un código de barras es leído, un sonido será reproducido y podrás ver los resultados del análisis, una descripción de lo que contiene el código de barras, y las distintas opciones ejecutar acciones. </p>
-<p> Si usted está teniendo problemas de escaneo, asegúrese de sujetar el teléfono estable. Si la cámara no puede enfocar, mueva el teléfono más lejos o más cerca del código de barras. </p>
+<p> Si usted está teniendo problemas de escaneo, asegúrese de sujetar el teléfono establemente. Si la cámara no puede enfocar, mueva el teléfono más lejos o más cerca del código de barras. </p>
```

Comment:
How about "mantener el teléfono estable?". I don't think I've read the word "establemente" before.